src/stdlib_io.fypp

```diff
@@ -142,56 +143,51 @@ contains
       allocate(d(max_rows_, ncol))
 
       do i = 1, skiprows_
-        read(s, *)
+        read(s, *, iostat=ios, iomsg=iomsg)
+        
+        if (ios/=0) then 
+           write(msgout,1) trim(iomsg),i,trim(filename) 
+           call error_stop(msg=trim(msgout))
```

Comment:
Is it still wise to use `call error stop(msg)` instead of `error stop msg`? If I remember well, `call error_stop` was introduced because some compilers did not support `error stop` yet. However, I don't think it is still the case now.

Reply:
Yes! I used it for consistency. I agree that `error stop` is a better solution. Here [15de15d](https://github.com/fortran-lang/stdlib/pull/877/commits/15de15d951aa988229dcf15e4b7c767d0ab74b75) is the update.